nbs/09_API/01_config.ipynb

```diff
@@ -213,7 +213,16 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Could you please re-run this with a token set up, or just manually remove the error output, so this doesn't appear in the docs?

Reply:
This is done.  I tried to fix it the correct way and get a GITHUB_TOKEN set up, but wasn't able to figure it out.  I'll leave that challenge for another day